contracts/base/OwnerManager.sol

```diff
@@ -13,7 +13,7 @@ contract OwnerManager is SelfAuthorized {
     address internal constant SENTINEL_OWNERS = address(0x1);
 
     mapping(address => address) internal owners;
-    uint256 ownerCount;
+    uint256 private ownerCount;
```

Comment:
Why is this specific storage variable private while the others are internal?

Reply:
Changed